lib/flowbber/plugins/sources/lcov.py

```diff
@@ -182,6 +182,18 @@ class LcovSource(Source):
     def declare_config(self, config):
         config.add_option(
             'directory',
```

Comment:
or we can change this key for something like "source", so the user can choose if he want to specify a directory or a tracefile directly, so we don't have to add another option "tracefile".

Reply:
Exactly. I will change it then.